CONTRIBUTING.md

```diff
@@ -179,7 +179,36 @@ virtualenv venv
 # Install openhtf into the virtualenv in dev mode.
 python setup.py develop
 ```
+### MacOS
 
+Install Xcode and Homebrew Mac. Once that is done: 
+
+```bash
+#Install dependent packages
```

Comment:
nit: A space between the # and the comment text makes the comment more readable.
nit+: I like full capitalization and punctuation on comments as if they're sentences.

The [Google Shell Style Guide](https://google.github.io/styleguide/shell.xml) doesn't explicitly state these nits, but all its examples are consistent with them.